portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,81 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+
+const images =
+    ['basketball.JPG', 'chess.JPG', 'crochet.JPG', 'smoothie.jpg', 'statefarm.JPG',
+    'tennis.JPG', 'UIL_state.JPG', "Uke.jpg"];
+
+const descriptions =
```

Comment:
instead of maintaining parallel arrays, group related data together in object form, e.g.

```js
const images = [
  {name: 'basketball.jpg', label: 'Playing basketball with the Tennis Intramural Team!'},
  // ...
];
```

Rationale is that it's easier to keep a single list in-sync, and you can pass around related data together instead of dealing with indices.

Reply:
Makes sense, I'll make the change!